Vagrantfile

```diff
@@ -8,11 +21,14 @@ Vagrant.configure("2") do |config|
     config.cache.enable :npm
   end
 
+  config.vm.box = vagrant_box
   config.vm.hostname = "girder"
+  config.vm.provider "virtualbox" do |virtualbox|
+    virtualbox.memory = 2048
+  end
 
-  # Disable port forwarding for ansible testing, since it makes running
-  # VMs in parallel harder.
-  unless ENV["ANSIBLE_TESTING"]
+  # Disable port forwarding for ansible testing, since it makes running VMs in parallel harder.
+  unless is_testing
     config.vm.network "forwarded_port", guest: 8080, host: 9080
```

Comment:
I'd love to see automatic port collision handled, even if the post_up_message is sub-optimal.  We could add `auto_correct: true` and change the post_up_message to something like `Girder is running at http://localhost:9080 or another port if there was a conflict`.  When there is a collision, you see the message like `Fixed port collision for 8080 => 9080. Now on port 2200.` flit by, which is hard to pick out from the noise.